lib/src/backends/memory/common.dart

```diff
@@ -4,9 +4,6 @@
 
 import 'package:file/src/common.dart' as common;
 
-/// The file separator.
-const String separator = '/';
```

Comment:
Is this a potentially breaking change or has this property never been exported outside of the lib?

Reply:
It's never been exported

Reply:
It was also just merged this morning and hasn't been published yet 🙂